views/sidecolumn.html

```diff
@@ -1,36 +1,91 @@
 <!-- Filters Section -->
 {{ define "sidecolumn" }}
-<div id="sidecolumn" class="mt-12 ml-12">
-    <p>{{ .Count }} Results</p>
-    <div class="flex flex-col space-y-2">
-        {{ range .Filters }}
-        <div class="flex items-center space-x-2">
-            <input
-                    {{ if .Active }} checked {{ end }}
-                    type="checkbox"
-                    name="{{ .Name }}"
-                    id="{{ .Name }}"
-                    hx-post="/filter/{{.ID}}"
-                    hx-swap="innerHTML"
-                    hx-target="#results-container"
-                    hx-trigger="click"
-                    hx-select-oob="#sidecolumn"
-            >
-            <label for="{{ .Name }}" class="text-gray-700">{{ .Name }}</label>
+<aside id="sidecolumn" class="mx-6 mt-6 w-64 rounded-md bg-white">
+    {{ range $index, $filter := .Filters }}
+    <div id="accordion-{{ $index }}" class="border-b border-gray-200">
+      <!-- Accordion Header -->
+      <div id="accordion-heading-{{ $index }}" class="p-4 cursor-pointer bg-gray-100 hover:bg-gray-200">
+        <button
+        id="accordion-button-{{ $index }}"
+        aria-expanded="false"
+        aria-controls="accordion-body-{{ $index }}"
+        class="w-full"
+        onclick="toggleAccordion('{{ $index }}')">
+        <div class="flex items-center justify-between w-full spaxe-x-10">
+          <span class="text-black font-bold">{{ $filter.Category }}</span>
+          <div class="svg-wrapper w-4 h-4 flex-shrink-0">
+            <svg 
+              class="w-full h-full transform transition-transform duration-300" 
+              xmlns="http://www.w3.org/2000/svg" 
```

Comment:
download this into static/svg and use that rather than fetching this dynamically